CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## 0.1.1
+
+* Fix problem: can only installed on only python 3.9.6
```

Comment:
based on https://keepachangelog.com/en/1.0.0/
```suggestion
### Fixed:
Fixed a PyPI constraint where the package can only be installed on python ==3.9.6
```

Reply:
I miss Sasikala now. She always help me to fix such things .
Thanks .